package/lib/index.tests.js

```diff
@@ -543,13 +544,17 @@ describe('Operations (static)', function () {
       // A ∪ A = A.
       assert.isTrue(Set.union(a, a).equal(a))
 
-      // A ∪ U = U. // TODO even possible to test here? -> https://en.wikipedia.org/wiki/Russell%27s_paradox
+      // A ∪ U = U.
+      areEqual(u, Set.union(a, u))
```

Comment:
Russell's paradox says that there is no such thing as "the set of all sets".  I believe you are equating this to "the universe".

However, these are two separate concepts.  When working within a "universe", it is in fact *you* who gets to choose which universe you want to work within.  For example, I have chosen for my universe to be {1, 2, 3, 4, 5, 6, 7}.  All this really means is that all other sets (such as a, b, and c) cannot possibly contain elements outside of {1, 2, 3, 4, 5, 6, 7}.